tests/components/recorder/conftest.py

```diff
@@ -9,6 +9,30 @@
 from homeassistant.core import HomeAssistant
 
 
+def pytest_configure(config):
+    """Add custom skip_on_db_engine marker."""
+    config.addinivalue_line(
+        "markers",
+        "skip_on_db_engine(engine): mark test to run only on named DB engine(s)",
+    )
```

Comment:
We should maybe have a marker to only run tests on certain engines, as well as, or instead of, this marker which allows skipping tests on certain engines.